webapp/packages/plugin-object-viewer/src/ObjectPropertiesPage/ObjectInfoTab/ObjectProperties.tsx

```diff
@@ -38,12 +40,30 @@ export const ObjectProperties = observer(function ObjectProperties({
   }
 
   return styled(styles)(
-    <ColoredContainer overflow parent>
-      <Group gap large>
-        {dbObject.properties.map(v => (
-          <ObjectProperty key={v.id} objectProperty={v} small />
-        ))}
-      </Group>
+    <ColoredContainer overflow parent gap>
+      {isUncategorizedExists && (
+        <Group gap large>
+          <ObjectPropertyInfoFormNew
+            properties={dbObject?.properties}
+            category={null}
+            state={{}}
```

Comment:
seems like we have a case without the state, so why it provided as empty? also, the form is fully re-render each time `ObjectProperties ` is re-render, because of new state object creation